nomos-services/data-availability/network/src/opinion_aggregator.rs

```diff
@@ -28,12 +28,31 @@ pub enum OpinionError {
     Error(#[from] DynError),
 }
 
-pub struct Opinions {
-    pub session_id: SessionNumber,
-    pub new_opinions: BitVec,
-    pub old_opinions: BitVec,
+pub struct ActivityProof {
+    pub current_session: SessionNumber,
+    pub previous_session_opinions: BitVec<u8, Lsb0>,
+    pub current_session_opinions: BitVec<u8, Lsb0>,
 }
 
+impl ActivityProof {
+    pub fn to_metadata_bytes(&self) -> Vec<u8> {
+        let prev_bytes_len = self.previous_session_opinions.len().div_ceil(8);
+        let curr_bytes_len = self.current_session_opinions.len().div_ceil(8);
+        let total_size = 1 // version byte
+            + size_of::<SessionNumber>()
+            + prev_bytes_len
+            + curr_bytes_len;
+
+        let mut bytes = Vec::with_capacity(total_size);
+
+        bytes.push(0x01);
```

Comment:
Could this be a constant with a declarative name? Or link to a spec if it is defined in a spec.